docs/books/user-guide/managing-using-qdmanage.adoc

```diff
@@ -53,30 +53,62 @@ For example, the following command executes a query operation on a router, and t
 $ qdmanage query --type listener
 [
   {
-    "stripAnnotations": "both", 
-    "addr": "127.0.0.1", 
-    "multiTenant": false, 
-    "requireSsl": false, 
-    "idleTimeoutSeconds": 16, 
-    "saslMechanisms": "ANONYMOUS", 
-    "maxFrameSize": 16384, 
-    "requireEncryption": false, 
-    "host": "0.0.0.0", 
-    "cost": 1, 
-    "role": "normal", 
-    "http": false, 
-    "maxSessions": 32768, 
-    "authenticatePeer": false, 
-    "type": "org.apache.qpid.dispatch.listener", 
-    "port": "amqp", 
-    "identity": "listener/0.0.0.0:amqp", 
+    "stripAnnotations": "both",
+    "addr": "127.0.0.1",
+    "multiTenant": false,
+    "requireSsl": false,
+    "idleTimeoutSeconds": 16,
+    "saslMechanisms": "ANONYMOUS",
+    "maxFrameSize": 16384,
+    "requireEncryption": false,
+    "host": "0.0.0.0",
+    "cost": 1,
+    "role": "normal",
+    "http": false,
+    "maxSessions": 32768,
+    "authenticatePeer": false,
+    "type": "org.apache.qpid.dispatch.listener",
+    "port": "amqp",
+    "identity": "listener/0.0.0.0:amqp",
     "name": "listener/0.0.0.0:amqp"
   }
 ]
 ----
 
 For more information about `qdmanage`, see the {qdmanageManPageLink}.
 
+== Closing a connection
+
+If a consumer is processing messages too slowly, or has stopped processing messages without settling their deliveries, you can close the connection. By closing the connection, the "stuck" deliveries will be released, and they can be redelivered to a different consumer.
+
+.Procedure
+
+. Find the ID of the connection with the slow consumer.
++
+--
+This command lists the connections for a router in the router network:
+
+[options="nowrap"]
+----
+$ qdstat -c -r Router.A
+Connections
+id    host                        container                              role             dir  security     authentication  tenant
+==================================================================================================================================
+  2   127.0.0.1:5672                                                     route-container  out  no-security  anonymous-user
+  10  127.0.0.1:5001               Router.B                              inter-router     out  no-security  anonymous-user
+  12  localhost.localdomain:42972  161211fe-ba9e-4726-9996-52d6962d1276  normal           in   no-security  anonymous-user
+  14  localhost.localdomain:42980  a35fcc78-63d9-4bed-b57c-053969c38fda  normal           in   no-security  anonymous-user
+  15  localhost.localdomain:42982  0a03aa5b-7c45-4500-8b38-db81d01ce651  normal           in   no-security  anonymous-user
+----
+--
+
+. Close the connection by setting its `adminStatus` to `deleted`.
++
+[options="nowrap"]
+----
+$ qdmanage update --type=connection --id=12 adminStatus=deleted
```

Comment:
This might be obvious to Interconnect users (in which case, disregard it), but it wasn't to me:

What information in the previous output indicated that connection ID=12 had a slow consumer?